tools/update_repo.sh

```diff
@@ -558,6 +560,10 @@ EOF
         initiate_deb_metadata dists/$loop_dist/$component/binary-amd64/Packages
```

Comment:
amd64 is hardcoded here. Should not we initialize the `binary-arm64` directory for arm64 packages? It seems so according to the layout [here](https://mirror.yandex.ru/debian/dists/bullseye/main/).

Reply:
Fixed.